sequencing/vcf_pca/dxapp.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "vcf_pca",
   "title": "Calculate PCs from VCF",
-  "summary": "vcf_pca",
+  "summary": "Uses PLINK 1.9 and eigenstrat 6.0 to calculate principal components from VCF or PLINK bed/bim/fam files",
```

Comment:
Shouldn't there be a period at the end of this sentence? Run this through the dxapp.json linter